content/user/troubleshooting/new_php_debug_logs.md

```diff
@@ -0,0 +1,76 @@
+---
+title: New PHP Debug Logs 
+description: Newer PHP Debug Logs in Zen Cart 
+category: troubleshooting
+weight: 10
+---
+
+PHP is a dynamic language which is continually being updated.  And part of 
+the update process is sunsetting features, functions, syntax and other language elements: 
+
+- the language designers decide to change how something works
+- then the new way is introduced, and the old way is deprecated, 
+- then the old way is eliminated entirely. 
+
+The symptom you will see when this is an issue for you is that things 
+that used to work (even yesterday) suddenly do not.  What might have 
+happened in this case is that your hoster just changed your PHP version
+without warning.  Alternatively, perhaps you upgraded your version of 
+Zen Cart, but wound up using some old files.  
+
+Each version of Zen Cart (and each mod) 
+was developed for the version of PHP that was
+prevalent at the time, and may not be compatible with newer PHP versions.
+See [this article](/user/first_steps/server_requirements/#php-version) for the mapping of Zen Cart 
+version to PHP version. 
+
+Let's look at a specific example of how this can happen.  These examples will
+appear dated in a few years,  but the principles remain the same - a 
+language change was made that requires a change to your files. 
+
+Suppose you are using Zen Cart 1.5.4 with PHP 5.6, and it's working fine.  Then you upgrade to Zen Cart 1.5.6 and move to PHP 7.1. 
+
+At checkout time, you get a fatal error: 
+
+```
+PHP Parse error: syntax error, unexpected '[', expecting ',' or ';' in .../includes/modules/payment/linkpoint_api.php on line 320.
```

Comment:
Might be good to find a different example since this module is defunct now. No point in having any references to it in current docs.

Reply:
I think PayPal had the same issue ... I will look ...

Reply:
Yep.  Updated.